### PR TITLE
Adds 4.5 READMEs, and makes the files match up to the versions in the…

### PR DESCRIPTION
**Pull request by sandersn**
… spec

As requested by @orta

### CODE REVIEW

File: deploy/readmes/audioworklet.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ To use `@types/audioworklet` you need to do two things:
 
 That's all. 
 
+If you'd like to ensure that the DOM types are never accidentally included, you can use [@orta/types-noop]()https://www.npmjs.com/package/@orta/type-noops) in TypeScript 4.5+.
```
Comment (HolgerJeromin) [inline deploy/readmes/audioworklet.md:21]:
```suggestion
If you'd like to ensure that the DOM types are never accidentally included, you can use [@orta/types-noop](https://www.npmjs.com/package/@orta/type-noops) in TypeScript 4.5+.
```

File: deploy/readmes/serviceworker.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ To use `@types/serviceworker` you need to do two things:
 
 That's all. 
 
+If you'd like to ensure that the DOM types are never accidentally included, you can use [@orta/types-noop]()https://www.npmjs.com/package/@orta/type-noops) in TypeScript 4.5+.
```
Comment (HolgerJeromin) [inline deploy/readmes/serviceworker.md:21]:
```suggestion
If you'd like to ensure that the DOM types are never accidentally included, you can use [@orta/types-noop](https://www.npmjs.com/package/@orta/type-noops) in TypeScript 4.5+.
```

File: deploy/readmes/sharedworker.md [hunk 3]
```diff
@@ -11,13 +11,14 @@ This package contains type definitions which will set up the global environment
...
 1. Install the dependency: `npm install @types/sharedworker --save-dev`, `yarn add @types/sharedworker --dev` or `pnpm add @types/sharedworker --dev`.
+
 1. Update your [`tsconfig.json`](https://www.typescriptlang.org/tsconfig) to avoid clashing with the DOM APIs. There are two cases to consider depending on if you have `lib` defined in your `tsconfig.json` or not.
 
     1. **Without "lib"** - You will need to add `"lib": []`. The value you want to add inside your lib should correlate to your [`"target"`](https://www.typescriptlang.org/tsconfig#target). For example if you had `"target": "es2017"`, then you would add `"lib": ["es2017"]`
     1. **With "lib"**  - You should remove `"dom"`.
 
-That's all. 
 
+If you'd like to ensure that the DOM types are never accidentally included, you can use [@orta/types-noop]()https://www.npmjs.com/package/@orta/type-noops) in TypeScript 4.5+.
```
Comment (HolgerJeromin) [inline deploy/readmes/sharedworker.md:21]:
```suggestion
If you'd like to ensure that the DOM types are never accidentally included, you can use [@orta/types-noop](https://www.npmjs.com/package/@orta/type-noops) in TypeScript 4.5+.
```

**Review by HolgerJeromin — CHANGES_REQUESTED** [review]
Some links are broken

**Comment by orta** [comment]
Thanks @HolgerJeromin!

**Comment by HolgerJeromin** [comment]
@orta 
The PR was merged in the "buggy" state.

**Comment by HolgerJeromin** [comment]
This were many comments in a wrapping review. 
Perhaps you needed to approve all of them (as you did) but then commit all changes in a bunch (this is the work flow of azure devops at least).

**Comment by orta** [comment]
Thanks, I manually pushed that change to main

**Pull request merged**